crates/typos-cli/src/default_types.rs

```diff
@@ -112,7 +112,6 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("js", &["*.js", "*.jsx", "*.vue"]),
     ("json", &["*.json"]),
     ("jsonl", &["*.jsonl"]),
-    ("julia", &["*.jl"]),
```

Comment:
@LilithHafner would it be more idiomatic for the file type to be "julia" or "jl"?  Since we already refer to most files by their extension, rather than their full name, I went with `jl`.

Note this list was originally taken from ripgrep.  No idea why burntsushi made the decisions they did.